### DISCUSSION TITLE
Can i help to test TS101?

### DISCUSSION THREAD

**Discussion by XaocuHKa** [Ralim/IronOS#1775] [General]
Cooked my TS100 by supplying 40v on it (somebody interested in a body?), got TS101 and was SO upset IronOS is not available for it yet. Seen there was a PR a few weeks ago to support new HW. Can i help to test it?

**Comment by ia** [comment]
Hello!

> got TS101 and was SO upset IronOS is not available for it yet. Seen there was a PR a few weeks ago to support new HW.

Don't worry - _it is available_ but just not in Release builds yet since the support was added some time after previous stable release.

> Can i help to test it?

Sure! Any help is always welcome! You can always download & flash the latest fresh `dev` build for your `TS101` somewhere [here](https://github.com/Ralim/IronOS/actions/workflows/push.yml?query=branch%3Adev+event%3Apush+is%3Asuccess) - this is the dynamic link with the list of latest builds. So:
- just click on the latest green link
- scroll-down to `Artifacts` on the bottom of the page
- download `TS101` zip package
- unpack it
- pick binary which suits you best with language & type (dfu, hex or bin)
- flash it using the method which works for you

I just asked Ralim recently about [support of `TS101` & `S60`](https://github.com/Ralim/IronOS/discussions/1759#discussioncomment-6536230) myself: the only one thing which should be fixed is proper scaling of OLED since the display in `TS101` larger than the one in `TS100`. From the discussion above:

> the OLED currently only draws in top left 1/3rd until I finish all the OLED refactoring. (So its just cosmetic and 1000% fixable when I have time).

Hope it helped.

**Reply by XaocuHKa** [reply]
Spasibo! I'm flashing it tonight, i appreciate spending time to write all that, helps A LOT. Thank you
